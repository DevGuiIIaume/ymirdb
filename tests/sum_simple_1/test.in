set a 10
sum a
pop a
sum a
set a -1 -5 -10 3 0 10
sum a
append a 5
sum a
push a -1
sum a
bye
