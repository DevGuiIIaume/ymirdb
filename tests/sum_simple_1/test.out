> ok

> 10

> 10

> 0

> ok

> -3

> ok

> 2

> ok

> 1

> bye
